bqrole/project.go

```diff
@@ -63,8 +64,9 @@ func grantProjectRole(project, user, role string) error {
 		return fmt.Errorf("failed to fetch current policy: error %s", err)
 	}
 
-	if hasProjectRole(*policy, user, role) {
-		return fmt.Errorf("%s already has a role: %s, project: %s", user, role, project)
```

Comment:
This is why a process stops in the middle. Fix it by returns nil and log the event.